app/src/main/java/com/yeonae/chamelezone/view/course/CourseTabFragment.kt

```diff
@@ -33,13 +34,10 @@ class CourseTabFragment : Fragment(), CourseContract.View {
         }
     }
 
-    override fun showCourseList(courseList: List<CourseItem>) {
-        courseTabRvAdapter.addData(courseList)
+    override fun showCourseList(item: List<CourseItem>) {
```

Comment:
```suggestion
    override fun showCourseList(itemList: List<CourseItem>) {
```
itemList, items 등으로 변경이 필요해보입니다.

Reply:
수정했습니당